### DISCUSSION TITLE
Adding support for SVCB/HTTPS records types

### DISCUSSION THREAD

**Discussion by webtroter** [TechnitiumSoftware/DnsServer#434] [Ideas]
Hi,

So today I was reading about QUIC/HTTP3 and I learned about some new DNS Records Types :

- SVCB (Service Binding)
- HTTPS

These allow a client to know if the endpoint supports HTTPS (and/or HTTP2 and HTTP3/QUIC) without sending a packet to that server.

It is still being drafted by the IETF, but it has already been implemented in some servers/service (like Cloudflare and Apple devices)
[Service binding and parameter specification via the DNS (DNS SVCB and HTTPS RRs) ](https://www.ietf.org/archive/id/draft-ietf-dnsop-svcb-https-10.html)

**Comment by ShreyasZare** [comment]
Thanks for the post. Yes, I am tracking the IETF draft on their mailing list. It should soon get standardized. The support will be added soon for zones. 

The DNS server can resolve any kind of records that are unknown to it so when using it as recursive resolver, it will be able to return both of these records to the clients requesting them.
